Standards/CCSMB-15.md

```diff
@@ -0,0 +1,60 @@
+# _CCSMB-15_: Wide Printer Format and Wide Printer Format Array
+
+_Author: znepb_  
+_Version_: 1.0.0  
+_Last revised:_  
+The words MUST, MUST NOT, SHOULD, SHOULD NOT, and MAY are defined in [RFC 2119](https://www.rfc-editor.org/rfc/rfc2119).
+
+## Rational
+
+Wide Printer Format (2dj) is a file format used by [SwitchCraft peripherals](https://github.com/SwitchCraftCC/sc-peripherals) to print poster files. These files contain information to print onto these posters, each pixel being a different color similar to map files. Wide Format Print (A) (2dja), is similar to 2dj.
+
+## 2dj
+
+### Format
+
+| Information |                     |
+| ----------- | ------------------- |
+| Version     | 1.0.0               |
+| Type        | Wide Printer Format |
+| MIME        | `image/2dj`         |
+| Extensions  | `.2dj`              |
+
+Wide Printer Format files MUST follow [RFC 8259](https://datatracker.ietf.org/doc/html/rfc8259), "The JavaScript Object Notation (JSON) Data Interchange Format." These files MUST also use the file extension `.2dj`.
+
+### Keys
+
+- `label`: This key SHOULD contain the name of the poster. It MUST be a string.
+- `tooltip`: This key MAY contain a tooltip displayed while hovering over the poster in the players inventory. It MUST be a string.
+- `palette`: MUST be `null` or an array and MUST NOT have more than 63 values. This defines the color palette the poster will use.
```

Comment:
Thought: common color palette format?

Reply:
Color palettes tend to always differ between images, because they are adjusted to give the best colors for each poster.

Reply:
My bad, I wasn't super clear in my wording :sweat_smile: I meant a standardized color palette, as I believe that palettes are defined across multiple standards. Feel free to ignore this, it was just a thought.